lib/shallow-populate.js

```diff
@@ -1,5 +1,6 @@
 const assert = require('assert')
-const { getByDot, setByDot } = require('feathers-hooks-common')
+const getByDot = require('lodash.get')
```

Comment:
These packages are not current with the regular `lodash` package.  It's a better practice to use the `lodash` package from npm.  You can use es6 imports and tree shaking to get rid of the extra weight.